recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -12,7 +12,8 @@ DEPENDS = " \
     libsoup-2.4 libwebp libxml2 libxslt virtual/egl virtual/libgles2 libepoxy libgcrypt \
 "
 
-inherit cmake pkgconfig perlnative pythonnative cmake_qt5
+inherit cmake pkgconfig perlnative pythonnative
+inherit ${@'cmake_qt5' if 'qt5-layer' in d.getVar('BBFILE_COLLECTIONS').split() else ''}
```

Comment:
Should the qtwpe PACKAGECONFIG option remain as-is? It tracks some qt deps recipes.

Reply:
I think it's OK as is. The default PACAKGECONFIG for the recipe doesn't include qtwpe, so it won't cause failures if you don't have meta-qt5. If override the default PACKAGECONFIG to add qtwpe but don't add the meta-qt5 layer, you'll get errors pretty quickly for the missing qt build time dependencies.